### PR TITLE
New site added, old site moved

### PR DESCRIPTION
**Pull request by chamkank**
Here's the new site, old site was moved to the "old website" folder.

### CODE REVIEW

File: css/css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* latin */
```
Comment (d4l3k) [inline css/css]: What's this file?

File: index.html [hunk 2]
```diff
@@ -2,220 +2,88 @@
...
+	<!-- <link rel="stylesheet" href="./style.css"> -->
 	<link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon">
 	<link rel="icon" href="/img/favicon.ico" type="image/x-icon">
 	<meta name="viewport" content="width=device-width,initial-scale=1, minimum-scale=1, maximum-scale=1"/>
-	<link href='http://fonts.googleapis.com/css?family=Oxygen:400,700,300' rel='stylesheet' type='text/css'>
+	<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+	<link href="https://fonts.googleapis.com/css?family=Quicksand:400,700" rel="stylesheet">
 	<link rel="icon" type="image/png" href="/favicon.png">
-	<script>
+	<!--<script>
```
Comment (d4l3k) [inline index.html]: any reason for removing the tracking codes?

File: index.html [hunk 3]
```diff
@@ -2,220 +2,88 @@
...
-		<h3>StormTalk #006: iOS in Swift</h3>
-		<br>
-		<span class="date">March 30th</span>
-		<span class="time">6:00PM</span>
-		<span class="location">MCLD 418, UBC Vancouver</span>
-		<br>
-		<div class="intro-button">
-			<a href="https://form.jotform.com/60878812975270">RVSP your spot.</a>
-			<a href="https://www.facebook.com/events/1587947218193620/">Facebook Event</a>
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```
Comment (d4l3k) [inline index.html]: Is jQuery really necessary for just form.js? Seems like a ton of dependencies for just a few functions.

File: form.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        if(input.val() == input.attr('title')){
+            $(this).removeClass('unfocused').val('');
+        }
+    });
+	
+    // When the user loses focus on an input    
+    inputs.blur(function(){
+        var input = $(this);
+        if(input.val() == ''){ // User has not placed text
+            input.val(input.attr('title')).addClass('unfocused');
```
Comment (d4l3k) [inline form.js]: Pretty sure you can just do the focused/unfocused styles with just CSS and `:focus`.

**Comment by chamkank** [comment]
All the changes have been made, let me know if there's anything else that needs to be done before the merge.

**Comment by chamkank** [comment]
Btw I can't merge it because I don't have write access, so if you could merge it that would be great.

**Comment by d4l3k** [comment]
Sorry, didn't realize that. I'll fix that soon

**Pull request merged**